From 6a53e6e03718da7473ecb086df2a51c4e03a1015 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Fri, 11 Dec 2020 15:04:03 +0530
Subject: [PATCH] Enable AIDL interface for Light HAL

Eanbling the AIDL interface for Light HAL

Tracked-On: OAM-95426
Signed-off-by: svenate <salini.venate@intel.com>
---
 light/aidl/default/lights-default.rc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/light/aidl/default/lights-default.rc b/light/aidl/default/lights-default.rc
index 687ec97dd1..4cdd038ad9 100644
--- a/light/aidl/default/lights-default.rc
+++ b/light/aidl/default/lights-default.rc
@@ -1,5 +1,5 @@
 service vendor.light-default /vendor/bin/hw/android.hardware.lights-service.example
     class hal
-    user nobody
-    group nobody
+    user system
+    group system
     shutdown critical
-- 
2.45.2

